### PR TITLE
[logtalk/it-it] Add Italian translation of the Logtalk tutorial]

### PR DESCRIPTION
**Pull request by pmoura**
- [*] Pull request title is prepended with `[language/lang-code]`
- [*] Pull request touches only one file (or a set of logically related files with similar changes made)
- [*] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [*] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [*] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: it-it/logtalk-it.html.markdown [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+X = 99
+true
+```
+
+La libreria Logtalk fornisce infine il supporto per combinare hook agli oggetti utilizzando diversi modi (ad esempio, definendo una pipeline di espansioni).
+
+
+# Maggiori informazioni
+
+Visita il [Sito web di Logtalk](http://logtalk.org) per maggiori informazioni.
```
Comment (vendethiel) [inline it-it/logtalk-it.html.markdown]: please add a (EN) tag to signify the link isn't in italian, please :).

Comment (pmoura) [inline it-it/logtalk-it.html.markdown]: Just to be clear, you mean something like `... [Sito web di Logtalk (EN)](...`?

Comment (pmoura) [inline it-it/logtalk-it.html.markdown]: Pushed.

**Comment by vendethiel** [comment]
Thank you very much! :)

**Comment by pmoura** [comment]
Just noticed on the website that syntax coloring for the Logtalk language is working :-) This italian version is not tagging the code blocks, however. I will submit an update later.

**Pull request merged**